roles/update/tasks/main.yml

```diff
@@ -14,6 +14,12 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Hello World
-  ansible.builtin.debug:
-    msg: "Hello World from update"
+- name: Run make openstack_update_run
+  ansible.builtin.include_role:
+    name: 'install_yamls_makes'
+    tasks_from: 'make_openstack_update_run'
+  vars:
+    make_openstack_update_run_params:
+      FAKE_UPDATE: false
+      VERSION: "0.0.2"
```

Comment:
We should be able to reach those parameters from the job definition and as well those:

```
CONTAINERS_NAMESPACE=${CONTAINERS_NAMESPACE:-podified-antelope-centos9}
CONTAINERS_TARGET_TAG=${CONTAINERS_TARGET_TAG:-current-podified}
FAKE_UPDATE=${FAKE_UPDATE:-false}
OPENSTACK_VERSION=${OPENSTACK_VERSION:-0.0.2}
TIMEOUT=${TIMEOUT:-300s}
```

Hum, and it seems the install_yaml [review](https://github.com/openstack-k8s-operators/install_yamls/pull/843) is  using OPENSTACK_VERSION in the install_yaml

So either new variables (maybe not the timeout one ...) for each those in the update role or would be best as they are kinda important variable and not related to deployment.

Maybe `CONTAINERS_TARGET_TAG` can be consolidated with what we have for `cifmw_repo_setup_promotion` in `cifmw_update_extras` in the job definition ?

Also look at [that comment in the ci-framework review](https://github.com/openstack-k8s-operators/ci-framework/pull/1963#pullrequestreview-2151312090)

WDYT?

Reply:
it is all passed from ci-framework to install_yaml as parameters 
 CONTAINERS_TARGET_TAG is cifmw_update_openstack_update_run_containers_target_tag